src/test/swagger/oas3.yaml

```diff
@@ -64,6 +64,7 @@ tags:
     description: Find out more about our store
     url: http://swagger.io
 paths:
+<<<<<<< HEAD
```

Comment:
it seems this is a git conflict changes.

Reply:
removed